app/build.gradle

```diff
@@ -1,9 +1,14 @@
+plugins {
+    id 'io.gitlab.arturbosch.detekt' version "1.0.0.RC6-3"
```

Comment:
Can't this be moved to the detekt.gralde file?

Reply:
I can check, tho I remember that gradle complained before that `plugins` needs to be topmost instruction